### PR TITLE
BNGP-3378: Add Bacs details to confirmation screen

### PR DESCRIPTION
**Pull request by sfitz42**

### CODE REVIEW

**Review by steve-dickinson — APPROVED** [review]
LGTM

**Review by steve-dickinson — APPROVED** [review]
LGTM

**Review by paulsimonandrews — CHANGES_REQUESTED** [review]
Just a quick update to replace `substr` with `substring`

File: packages/webapp/src/utils/helpers.js [hunk 1]
```diff
@@ -263,6 +263,8 @@ const getDeveloperEligibilityResults = session => {
 
 const formatAppRef = appRef => `${appRef.substr(0, 3)}-${appRef.substr(3, 3)} ${appRef.substr(6, 3)} ${appRef.substr(9, appRef.length)}`
 
+const formatSortCode = sortCode => `${sortCode.substr(0, 2)} ${sortCode.substr(2, 2)} ${sortCode.substr(4, 2)}`
```
[the remark this answers is not in the document]
Comment (sfitz42) [reply packages/webapp/src/utils/helpers.js]: Updated to use substring

File: packages/webapp/src/utils/config.js [hunk 2]
```diff
@@ -5,3 +5,7 @@ export const REDIS_TLS = process.env.REDIS_TLS || false
 export const REDIS_PASSWORD = process.env.REDIS_PASSWORD || ''
 export const SESSION_COOKIE_PASSWORD = process.env.SESSION_COOKIE_PASSWORD || 'the-password-must-be-at-least-32-characters-long'
 export const KEEP_ALIVE_TIMEOUT_MS = process.env.KEEP_ALIVE_TIMEOUT_MS || undefined
+export const BACS_ACCOUNT_NAME = process.env.BACS_ACCOUNT_NAME || 'Natural England'
+export const BACS_SORT_CODE = process.env.BACS_SORT_CODE || '123456'
+export const BACS_ACCOUNT_NUMBER = process.env.BACS_ACCOUNT_NUMBER || '12345678'
+export const BACS_SWIFT_CODE = process.env.BACS_SWIFT_CODE || 'ABCDEF2G'
```
Comment (KiranVarma02) [inline packages/webapp/src/utils/config.js:11]: just asking for my understanding - is it okay to have defaults in a production app?

Comment (steve-dickinson) [reply packages/webapp/src/utils/config.js:11]: Its ok to have defaults

File: packages/webapp/src/utils/helpers.js [hunk 3]
```diff
@@ -263,7 +263,7 @@ const getDeveloperEligibilityResults = session => {
 
 const formatAppRef = appRef => `${appRef.substr(0, 3)}-${appRef.substr(3, 3)} ${appRef.substr(6, 3)} ${appRef.substr(9, appRef.length)}`
 
-const formatSortCode = sortCode => `${sortCode.substr(0, 2)} ${sortCode.substr(2, 2)} ${sortCode.substr(4, 2)}`
+const formatSortCode = sortCode => `${sortCode.substring(0, 2)} ${sortCode.substring(2, 4)} ${sortCode.substring(4, 7)}`
```
Comment (paulsimonandrews) [inline packages/webapp/src/utils/helpers.js:266]: Last bit should be `${sortCode.substring(4, 6)` not 7 (although this technically works as it's just going to the end of the string with 7)

**Review by paulsimonandrews — CHANGES_REQUESTED** [review]
Tiny change

File: packages/webapp/src/utils/helpers.js (hunk 3 above)
Comment (sfitz42) [reply packages/webapp/src/utils/helpers.js]: Fixed

**Review by paulsimonandrews — APPROVED** [review]

**Comment by shaundefra** [comment]
Tested locally https://eaflood.atlassian.net/browse/BNGP-3378

**Pull request merged**